traffic_ops/ort/traffic_ops_ort.pl

```diff
@@ -2652,6 +2658,23 @@ sub scrape_unencode_text {
 	return \@lines;
 }
 
+sub scrape_canned_comments {
+	my $linesin = $_[0];
+
+	my @linesout;
+
+	foreach my $line (@$linesin) {
+		if ( $line =~ m/^\#/ ) {
+			if ( $line =~ m/DO NOT EDIT - Generated for / || $line =~ m/$header_comment/ || $line =~ m/TRAFFIC OPS NOTE\:/ || $line =~ m/^##OVERRID.*##/ ) {
```

Comment:
Question: why don't we skip all comments?

Reply:
This function applies not just to ats config files but also to other type files that may not use '#' as a comment convention.  This is part of the hackery that is ORT.  Honestly I'm a bit scared about increasing scope of the changes.  You'll notice code that's left over is now checking for things that have already been stripped.  How much refactoring should be done?

Reply:
Ah, got it. Agreed.